backend/src/app/migrations/sql/0067-add-team-invitation-table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE team_invitation (  
+  team_id uuid NOT NULL REFERENCES team(id) ON DELETE CASCADE,
+  email_to text NOT NULL,
+  role varchar(255) NOT NULL,
+  valid_until timestamptz NOT NULL,
+  created_at timestamptz NOT NULL DEFAULT clock_timestamp(),
+  updated_at timestamptz NOT NULL DEFAULT clock_timestamp(),
```

Comment:
I think is better use `now()` here. I know that we have used the `clock_timestamp()` previosly in other sites and it is inconsistent across the codebase, but for now, the correct approach is use the `now()` unless we really need the `clock_timestamp()` behavior.

Reply:
fixed